### ISSUE TITLE
PATH_INFO-based rewriting by default

### ISSUE THREAD

**Issue by franzliedke** [flarum/core#426]
Nginx servers will have to be configured properly anyway.

For Apache, this has the benefit of working well even when mod_rewrite is not available / enabled.

**Comment by tobscure** [comment]
@franzliedke Any updates on this and #514?

**Comment by franzliedke** [comment]
Yeah, I had made some progress, but it proved more difficult than I had anticipated. I'll definitely take a look at #514 this week.

**Comment by franzliedke** [comment]
I'll have to push this one back. Too complicated...
